### DISCUSSION TITLE
Welcome to tpt-oracle Discussions!

### DISCUSSION THREAD

**Discussion by tanelpoder** [tanelpoder/tpt-oracle#28] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by ursusca** [comment]
Hello,

My name is Michael. I'm an Oracle DBA but my interest is also Linux, other data technologies and automation. So far I'm a happy user of the scripts but I would be happy to contribute.

**Comment by Project-42** [comment]
Hi,
Victor here.
Working with Oracle Databases last 8-9 years and die hard fan of your scripts and content :)

**Comment by cpclaudio** [comment]
Hello all,

My name is Claudio, I started working with Oracle database in 2003 as junior DBA with 7.3.4 version migrating from development environment to infrastructure.

I had a opportunity to discover the tpt-oracle scripts in the middle of 2012, since then it has been help me a lot on my troubleshooting tasks and personally I think it is a valuable tool for the community.

Cheers,
